Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:jammy
+
+LABEL py_version='3.x'
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+VOLUME /dist
+
+RUN apt-get update && apt-get install -y \
+    curl \
+    libgdal-dev \
+    libhdf5-dev \
+    libnetcdf-dev \
+    libxml2-dev \
+    python3-dev \
+    python3-venv \
+    g++ \
+    cmake \
+    git \
+    unzip \
+    zip \
+&& rm -rf /var/lib/apt/lists/*
+
+# First fetch the code for MDAL and mdal-python
+
+WORKDIR /
+
+ARG MDALVERSION=release-1.0.3
+
+# using curl to download the zips is much faster than a simple git clone
+RUN curl -Lo MDAL.tar.gz https://github.com/lutraconsulting/MDAL/archive/refs/tags/${MDALVERSION}.tar.gz && \
+    tar -xzf MDAL.tar.gz && \
+    rm MDAL.tar.gz && \
+    mv MDAL-${MDALVERSION} MDAL
+
+# Build MDAL
+WORKDIR /MDAL/build
+RUN cmake -DCMAKE_BUILD_TYPE=Rel -DENABLE_TESTS=ON .. && make && cmake --install .
+
+# Avoid issues with upgrading an apt-managed pip.
```

Comment:
Are there any? I mostly do `pip install --upgrade pip wheel`

Reply:
Tbh I copied that and assumed that if it was an issue in a different docker image, it might be here as well. I'll get rid of it.